docs/source/operator_reference.rst

```diff
@@ -4,23 +4,49 @@
 Operators
 =========
 
-Loading operators.
-
-.. raw:: html
-
-    <iframe
-      src="_static/dpf.html#CPython"
-      style="
-        position: fixed;
-        top: 36px;
-        bottom: 0px;
-        right: 0px;
-        width: 100%;
-        border: none;
-        margin: 0;
-        padding: 0;
-        overflow: hidden;
-        z-index: 1000;
-        height: 100%;
-      ">
-    </iframe>
+Starting with the DPF server for Ansys 2023 R2, operator availability is
```

Comment:
Just double checking if it is expected to mention 2023 R2. Or should it be 2023 R1 or 2022 R2?
```suggestion
Starting with the DPF server for Ansys 2023 R1, operator availability is
```

Reply:
@MaxJPRey The "DPF Server" is indeed going to be a pre-release of 2023 R2.